--- build~/src/utils/cas/configfile.c	2010-03-22 01:12:34.000000000 +0800
+++ build/src/utils/cas/configfile.c	2010-03-22 01:19:13.000000000 +0800
@@ -44,7 +44,7 @@
 		"# font_size - size the font\n",
 		"# source_image - image where the text will be writen\n",
 		"# *_line - x,y,[1/0] enabled or disabled\n\n",
-		"font /usr/share/fonts/corefonts/times.ttf\n",
+		"font /usr/share/fonts/truetype/ttf-dejavu/DejaVuSerif.ttf\n",
 		"font_size 10.5\n",
 		"source_image /usr/share/cas/stat.png\n",
 		"first_line 23,17,1\n",
